### ISSUE TITLE
Roundtrip validation failed while unpacking msapp

### ISSUE THREAD

**Issue by p3pijn** [microsoft/PowerApps-Language-Tooling#282]
Pac runs into errors while unpacking this Microsoft sample app: https://github.com/microsoft/PowerApps-Samples/blob/master/Embed/Canvas/samples/PowerAppsPlayerSdk/Player%20SDK%20sample.msapp

```
$ ./pac canvas unpack --sources "/sources/playersdk" --msapp "/msapps/Player SDK sample.msapp"
UnPack from /msapps/PlayerSDKsample.msapp-->/sources/playersdk
  source format version: 0.23
Error   PA3011: Roundtrip validation on unpack failed. This is always a bug, please file an issue at https://github.com/microsoft/PowerApps-Language-Tooling
Error   PA3011: Roundtrip validation on unpack failed. This is always a bug, please file an issue at https://github.com/microsoft/PowerApps-Language-Tooling
Microsoft PowerPlatform CLI
Version: 1.7.2+gd1a5b9b

Error: There were errors.
```

**Comment by zhaparoff** [comment]
Have the same issue once updated my canvas app version from 3.21064.25 to 3.21071.26

**Comment by MikeStall** [comment]
Diagnosed. What's happening is unpacking is adding a new flag in the template:
    "FirstParty": false,
    
The sample is older ("HeaderLastSavedDateTimeUTC": "08/20/2019 01:29:03",)

**Comment by MikeStall** [comment]
fixed in #296
